pkg/label/label_test.go

```diff
@@ -48,56 +31,44 @@ func TestNilMap(t *testing.T) {
 	}
 }
 
-func TestRemoveLabelsWithKRMPrefix(t *testing.T) {
-	labels := map[string]string{
-		"test.io/foo": "bar",
-		"key1":        "val1",
-	}
-	result := label.RemoveLabelsWithKRMPrefix(labels)
-	expectedResult := map[string]string{
-		"key1": "val1",
-	}
-	if !reflect.DeepEqual(result, expectedResult) {
-		t.Errorf("results mismatch: got '%v', want '%v'", result, expectedResult)
-	}
-}
-
 func TestNewGCPLabelsFromK8sLabelsBasicMap(t *testing.T) {
 	labels := map[string]string{
 		"key1":        "val1",
 		"key2":        "val2",
 		"test.io/foo": "bar",
 	}
-	result := label.NewGCPLabelsFromK8SLabels(labels, label.GetDefaultLabels())
-	expectedResult := map[string]interface{}{
+	result := label.NewGCPLabelsFromK8sLabels(labels)
+	expectedResult := map[string]string{
 		"key1":            "val1",
 		"key2":            "val2",
 		"managed-by-cnrm": "true",
 	}
 	if !reflect.DeepEqual(result, expectedResult) {
 		t.Errorf("results mismatch: got '%v', want '%v'", result, expectedResult)
 	}
-	result = label.NewGCPLabelsFromK8SLabels(labels)
-	expectedResult = map[string]interface{}{
-		"key1": "val1",
-		"key2": "val2",
+	result = label.NewGCPLabelsFromK8sLabels(labels)
```

Comment:
Nit: Looks like  all the rest are duplicated test cases. Could you remove them?

Reply:
Ah, yeah, good catch! Thanks. Fixed